org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaLanguageServerPlugin.java

```diff
@@ -126,6 +127,9 @@ public void start(BundleContext bundleContext) throws BackingStoreException {
 
 		// Set the ID to use for preference lookups
 		JavaManipulation.setPreferenceNodeId(PLUGIN_ID);
+		// Initializes default preferences
+		preferenceManager.getPreferences().setImportOrder(Preferences.JAVA_IMPORT_ORDER_DEFAULT);
```

Comment:
I'd call preferences.setImportOrder(Preferences.JAVA_IMPORT_ORDER_DEFAULT) in preferenceManager constructor.
Even then that'd still feel dirty. A little less dirty, but dirty anyway (dirty because it mutates the plugin preferences)